rails_docker/Gemfile.txt

```diff
@@ -3,7 +3,7 @@ ruby '2.6.2'
 
 # Backend
 gem 'rails', '5.2.3' # Latest stable
-gem 'pg' # Use Postgresql as database
+gem 'pg', '1.1.4' # Use Postgresql as database
```

Comment:
Is there a strong reason to fix the database version? If we were to start a new project, using the latest pg gem and database is preferred.

Reply:
Will retry without the version parameter.